api/bash/migrate-repos-in-org.sh

```diff
@@ -0,0 +1,391 @@
+      --header "authorization: Bearer $GITHUB_TOKEN" \
+      --header 'content-type: application/json' \
+      --header 'application/vnd.github.hellcat-preview+json' \
+      --data  \"{\"permission\": \"pull\"}\"
+
+   ########################
+   # Check for any errors #
+   ########################
+   if [ $? -ne 0 ]; then
+      echo "Error! Failed to set permission"
+      ((ERROR_COUNT++))
+   fi
+
+   ####################################
+   # Update the Write Permission Team #
+   ####################################
+   echo "-----------------------------------------------------"
+   echo "Setting Write Team Permissions"
+   curl -s --request PUT \
+      --url $GITHUB_URL/teams/$WRITE_TEAM/repos/$MASTER_ORG/$REPO_TO_UPDATE_PERMS \
+      --header "authorization: Bearer $GITHUB_TOKEN" \
+      --header 'content-type: application/json' \
+      --header 'application/vnd.github.hellcat-preview+json' \
+      --data  \"{\"permission\": \"push\"}\"
+
+   ########################
+   # Check for any errors #
+   ########################
+   if [ $? -ne 0 ]; then
+      echo "Error! Failed to set permission"
+      ((ERROR_COUNT++))
+   fi
+
+   ####################################
+   # Update the Admin Permission Team #
+   ####################################
+   echo "-----------------------------------------------------"
+   echo "Setting Admin Team Permissions"
+   curl -s --request PUT \
+      --url $GITHUB_URL/teams/$ADMIN_TEAM/repos/$MASTER_ORG/$REPO_TO_UPDATE_PERMS \
+      --header "authorization: Bearer $GITHUB_TOKEN" \
+      --header 'content-type: application/json' \
+      --header 'application/vnd.github.hellcat-preview+json' \
+      --data  \"{\"permission\": \"admin\"}\"
+
+   ########################
+   # Check for any errors #
+   ########################
+   if [ $? -ne 0 ]; then
+      echo "Error! Failed to set permission"
+      ((ERROR_COUNT++))
+   fi
+}
+################################################################################
+#### Function GetOrigOrgRepos ##################################################
+GetOrigOrgRepos()
+{
+   ##############################
+   # Get response with all info #
+   ##############################
+   echo "-----------------------------------------------------"
+   echo "Gathering all repos from Original Organization:[$ORIG_ORG]"
+   ORIG_ORG_RESPONSE=$(curl -s --request GET \
+      --url $GITHUB_URL/orgs/$ORIG_ORG/repos \
+      --header "authorization: Bearer $GITHUB_TOKEN" \
+      --header 'content-type: application/json')
+
+   #######################################################
+   # Loop through list of repos in original organization #
+   #######################################################
+   echo "-----------------------------------------------------"
+   echo "Parsing repo names from Original Organization:"
+   for orig_repo in $(echo "${ORIG_ORG_RESPONSE}" | jq -r '.[] | @base64');
```

Comment:
This is what I am referring to earlier. You encode in base64 here for the for loop, later line 235 you decode. I don't think you are encoding in the earlier call with jq I commented on above ☝️